src/tree/SubscriptionTreeItem.ts

```diff
@@ -63,77 +53,38 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
-        this.cache = {
-            resourceGroups: [],
-            appResources: [],
-            treeMaps: {}
-        }
+    private async resolveResourceGroups(context: IActionContext) {
+        const client: ResourceManagementClient = await createResourceClient([context, this.subscription]);
+        this.cache.resourceGroups = await uiUtils.listAllIterator(client.resourceGroups.list());
     }
 
     public constructor(parent: AzExtParentTreeItem, subscription: ISubscriptionContext) {
         super(parent, subscription);
-        this.resetCache();
         this.registerRefreshEvents();
     }
 
     public hasMoreChildrenImpl(): boolean {
-        return !!this.cache.nextLink;
+        return false;
     }
 
-    public async loadMoreChildrenImpl(clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
-        try {
-            const focusedGroupId = await ext.context.workspaceState.get('focusedGroup') as string;
-            let focusGroupTreeItem: GroupTreeItemBase | undefined;
-            if (this._triggeredByFocusGroupSetting) {
-                focusGroupTreeItem = await this.tryGetFocusGroupTreeItem(focusedGroupId);
-                if (focusGroupTreeItem) {
-                    return [focusGroupTreeItem];
-                }
-            } else if (!this._triggeredByDefaultGroupBySetting) {
-                if (clearCache) {
-                    this.resetCache();
-                }
-
-                if (this.cache.appResources.length === 0) {
-                    const resources = await applicationResourceProviders[azureResourceProviderId]?.provideResources(this.subscription) ?? [];
+    public async loadMoreChildrenImpl(_clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
+        const resources = await applicationResourceProviders[azureResourceProviderId]?.provideResources(this.subscription) ?? [];
 
-                    // To support multiple app resource providers, need to use this pattern
-                    // await Promise.all(applicationResourceProviders.map((provider: ApplicationResourceProvider) => async () => this.rgsItem.push(...(await provider.provideResources(this.subscription) ?? []))));
+        // To support multiple app resource providers, need to use this pattern
+        // await Promise.all(applicationResourceProviders.map((provider: ApplicationResourceProvider) => async () => this.rgsItem.push(...(await provider.provideResources(this.subscription) ?? []))));
+        resources.forEach(item => ext.activationManager.onNodeTypeFetched(item.type));
+        this.cache.appResources = resources;
 
-                    resources.forEach(item => ext.activationManager.onNodeTypeFetched(item.type));
-                    this.cache.appResources = resources.map((resource: AppResource) => AppResourceTreeItem.Create(this, resource));
-                }
-                await this.createTreeMaps(clearCache, context);
+        await this.resolveResourceGroups(context);
```

Comment:
If the grouping is not resource groups, do we still need to resolve info about them?